test/integration/offchain/healthcheck.test.ts

```diff
@@ -15,6 +15,9 @@ const getSpecVersion = (actualResult: any) => {
 const getIpfsVersion = (actualResult: any) => {
   return actualResult?._body?.details?.ipfs?.detail?.version
 }
+const getIdentityVersion = (actualResult: any) => {
```

Comment:
not sure if version is being disclosed, we could maybe get from package.json since we are using as external lib (i mean identity service)